server/src/main/java/org/elasticsearch/action/admin/cluster/node/info/NodesInfoRequest.java

```diff
@@ -30,6 +31,7 @@ public final class NodesInfoRequest extends BaseNodesRequest<NodesInfoRequest> {
      * @param in A stream input object.
      * @throws IOException if the stream cannot be deserialized.
      */
+    @UpdateForV9 // should be unused now
```

Comment:
It is [used](https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/action/admin/cluster/node/info/TransportNodesInfoAction.java#L107), though? 🤔

Reply:
> I don't understand the use of @UpdateForV9, though. Could you elaborate? Is it just a tag for cleanup that hasn't been gotten to yet?

Yeah, it even has docs :)

https://github.com/elastic/elasticsearch/blob/71238981c8428c1a25af8e86a05a74d447993120/libs/core/src/main/java/org/elasticsearch/core/UpdateForV9.java#L17-L18

> It is [used](https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/action/admin/cluster/node/info/TransportNodesInfoAction.java#L107), though? 🤔

Yeah sorry that comment was written from the point of view of someone reviewing this code for the v9 branch where it would be unused. Clarified in [7281d66](https://github.com/elastic/elasticsearch/pull/109512/commits/7281d6636cc5162d4565fefaeefab22fb14b886c).